src/libs/actions/WelcomeActions.js

```diff
@@ -9,75 +8,42 @@ import ONYXKEYS from '../../ONYXKEYS';
 import NameValuePair from './NameValuePair';
 import CONST from '../../CONST';
 
-/* Flag for new users used to show welcome actions on first load */
-let isFirstTimeNewExpensifyUser = false;
-Onyx.connect({
-    key: ONYXKEYS.NVP_IS_FIRST_TIME_NEW_EXPENSIFY_USER,
-    callback: val => isFirstTimeNewExpensifyUser = val,
-});
-
-const allReports = {};
-Onyx.connect({
-    key: ONYXKEYS.COLLECTION.REPORT,
-    callback: (val, key) => {
-        if (!val || !key) {
-            return;
-        }
-
-        allReports[key] = {...allReports[key], ...val};
-    },
-});
-
-const allPolicies = {};
-Onyx.connect({
-    key: ONYXKEYS.COLLECTION.REPORT,
-    callback: (val, key) => {
-        if (!val || !key) {
-            return;
-        }
-
-        allPolicies[key] = {...allPolicies[key], ...val};
-    },
-});
-
 /**
  * Shows a welcome action on first login
  *
  * @param {Object} params
  * @param {Object} params.routes
- * @param {Function} params.hideCreateMenu
+ * @param {Function} params.showCreateMenu
  */
```

Comment:
I think we should document the rest of the new params here

```suggestion
 * @param {Function} params.showCreateMenu
 * @param {Boolean} params.isFirstTimeNewExpensifyUser
 * @param {Array} params.allReports
 * @param {Array} params.allPolicies
 */
```